js/utilities.js

```diff
@@ -437,6 +437,50 @@ var ERMrest = (function(module) {
     };
 
     /**
+     * given sort object, get the string modifier
+     * @param {Object[]} sort [{"column":colname, "descending":boolean}, ...]
+     * @return {string} string modifier @sort(...)
+     * @private
+     */
+    module._getSortModifier = function(sort) {
```

Comment:
My suggestions should make it unnecessary for this function `_getSortModifier` and the next one `_getPagingModifier` to be properties of the `module`. They could just become private functions within the `parser.js` closure.

The only reason to have them then would be if you want to write unit tests for them (I don't personally think its necessary to have unit tests for small utility functions).

But even then, the right place for these would to put them in `parser.js` not in `utilities.js`.